### ISSUE TITLE
Error: FeatureSwitchContext was not correctly initialized

### ISSUE THREAD

**Issue by danieljimenez0255** [andrewsuzuki/react-twitter-widgets#40]
Hello, so I am using the Twitter API where I retrieve the users profile info and display it alongside their tweets. And I am using the Tweet component where I pass in all the tweet ids to display each Tweet. And it is able to display the tweets properly. And I am using the renderError prop just in case their is an error and I am also using the onLoad prop. The way I am using the onLoad prop is that while its loading it shows an h1 element saying the tweet is loading, but as soon as the actual tweet loads in, it sets this state I set up to true which replaces the h1 element with the Tweet. And it is able to do so successfully, but for some reason I am getting this weird error, and it is coming from the react-twitter-widgets source code. Here is an image of what I am talking about:
![Screenshot 2020-11-19 111637](https://user-images.githubusercontent.com/42664403/99713036-b9f10080-2a58-11eb-99f1-553883ce658c.png)

Would you know what could be causing this issue and how it would be able to be fixed?

**Comment by andrewsuzuki** [comment]
Not sure. Can you print the stack trace without minification? That error isn't coming directly from this package (search for FeatureSwitchContext), though it could be coming from a dep.

That isn't really the intended use of onLoad, but if you want to continue using it that way, I would ensure that the Tweet is still being rendered by React regardless of whether it's been loaded. To do this, I'd probably wrap it in a div and set display:none.

**Comment by danieljimenez0255** [comment]
Oh okay, I will wrap the h1 in the dev and do it that way. Thank you for the help and fast response!

**Comment by danieljimenez0255** [comment]
Oh okay I just realized it was because of the way I am rendering tweets. So I stored all the individual tweet info inside an array and I am mapping through it and passing in the tweet id into the Tweet component and it renders all of them on the page. And all the tweets render because I can see all of them when the page loads it just gives me that weird error. But if I only render one Tweet it doesn't give me that error anymore
